### DISCUSSION TITLE
Nation wide change in email address domain

### DISCUSSION THREAD

**Discussion by nathanielchanwj** [community/community#52948] [GitHub Education] [answered]
Hi, I am currently studying at Anglo-Chinese School (Barker Road) and have been applying for Github Edu for quite some time but I keep getting rejected as it states by email should end with @students.acsbr.org  But since 2021, the Singapore government has made all Singapore secondary schools emails to be issued by the government and now end with @students.edu.sg It would really help if you could change this as Github Edu would really benefit me.
<img width="642" alt="image" src="https://user-images.githubusercontent.com/61648759/232272068-e3ddab3f-d603-43e5-acf4-8731edc53408.png">

**Comment by sascharo** [comment]
I have an @e.ntu.edu.sg domain email address, and it doesn't help. In previous years, I had no issue, but starting this year, the domain is not accepted by GitHub anymore.

**Reply by sascharo** [reply, marked as answer]
With manual paperwork, mine finally got approved.

**Reply by nathanielchanwj** [reply]
yup mine finally got approved too
